### PR TITLE
Added a Makefile and better docker support

### PR DESCRIPTION
**Pull request by HeahDude**
Démarrer le projet :
```bash
make
```

La commande au-dessus execute toutes celles en-dessous (hormis "watch" et "test").

Builder et lancer docker :
```bash
make boot
```

Installer les dépendances PHP et JS :
```bash
make install
```

Générer les assets :
```bash
make assets
```

Générer les assets en temps réel :
```bash
make watch
```

Lancer le serveur PHP :
```bash
make run
```

Lancer les tests :
```bash
make test
```

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -16,3 +19,11 @@ services:
...
             - "9025:1025"
             - "9080:1080"
+
+    tools:
+        build: ./docker/tools
+        depends_on:
+            - mysql
+        volumes:
+            - .:/www
+            - var:/www/var
```
Comment (salahm) [inline docker-compose.yml]: why do you need this  ?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,22 @@
+TOOLS = docker-compose run --rm tools
+CONSOLE = ./bin/console
```
Comment (salahm) [inline Makefile]: why aren't you using docker for the console also ?

File: docs/1. Installer le projet en local.md [hunk 3]
```diff
@@ -19,10 +19,18 @@ n'hésitez pas à nous poser la question [sur Slack](https://slack.en-marche.fr)
...
  
-- MariaDB 5.5+
 - PHP 7.0+
-- Composer
-- nodeJS/npm/yarn
+- MariaDB 5.5+ (ou Docker/Docker compose)
+- Composer (ou Docker/Docker compose)
+- nodeJS/npm/yarn (ou Docker/Docker Compose)
+
+Si vous avez Docker, installer et lancer le projet simplement avec :
```
Comment (tgalopin) [inline docs/1. Installer le projet en local.md]: installez, lancez

File: docker-compose.yml (hunk 1 above)
Comment (HeahDude) [inline docker-compose.yml]: Good catch!

File: Makefile (hunk 2 above)
Comment (HeahDude) [inline Makefile]: Because it would launch the PHP server inside tools container :/

File: docs/1. Installer le projet en local.md (hunk 3 above)
Comment (HeahDude) [inline docs/1. Installer le projet en local.md]: Thanks!

**Comment by HeahDude** [comment]
@salahm @tgalopin Comments addressed.

File: Makefile (hunk 2 above)
Comment (salahm) [inline Makefile]: then, we may use a real webserver in docker

Comment (HeahDude) [inline Makefile]: Yes, later, this was a quick enhancement, not a Docker integration which is out of scope.

**Pull request merged**